pkg/cmd/version/version.go

```diff
@@ -30,39 +30,40 @@ type VersionOptions struct {
 }
 
 func NewCmdVersion(commonOpts *opts.CommonOptions) *cobra.Command {
-	options := &VersionOptions{
+	vOpts := &VersionOptions{
 		CommonOptions: commonOpts,
 	}
 
 	cmd := &cobra.Command{
 		Use:   "version",
 		Short: "Print the version information",
 		Run: func(cmd *cobra.Command, args []string) {
-			options.Cmd = cmd
-			options.Args = args
-			err := options.Run()
+			vOpts.Cmd = cmd
+			vOpts.Args = args
+			err := vOpts.Run()
 			helper.CheckErr(err)
 		},
 	}
 	/*
 		cmd.Flags().BoolP("client", "c", false, "Client version only (no server required).")
 		cmd.Flags().BoolP("short", "", false, "Print just the version number.")
 	*/
-	cmd.Flags().MarkShorthandDeprecated("client", "please use --client instead.")
-	cmd.Flags().BoolVarP(&options.HelmTLS, "helm-tls", "", false, "Whether to use TLS with helm")
-	cmd.Flags().BoolVarP(&options.NoVersionCheck, "no-version-check", "n", false, "Disable checking of version upgrade checks")
-	cmd.Flags().BoolVarP(&options.NoVerify, "no-verify", "", false, "Disable verification of package versions")
-	cmd.Flags().StringVarP(&options.Namespace, "namespace", "", "", "The namespace to use to look for currently installed platform version")
+	_ = cmd.Flags().MarkShorthandDeprecated("client", "please use --client instead.")
+	cmd.Flags().BoolVarP(&vOpts.HelmTLS, "helm-tls", "", false, "Whether to use TLS with helm")
+	cmd.Flags().BoolVarP(&vOpts.NoVersionCheck, "no-version-check", "n", false, "Disable checking of version upgrade checks")
+	cmd.Flags().BoolVarP(&vOpts.NoVerify, "no-verify", "", false, "Disable verification of package versions")
+	cmd.Flags().StringVarP(&vOpts.Namespace, "namespace", "", "", "The namespace to use to look for currently installed platform version")
 	return cmd
 }
 
 func (o *VersionOptions) Run() error {
 	ns := o.Namespace
+	fmt.Println("Namespace: ", ns)
```

Comment:
I guess that's a leftover from debugging?

Reply:
Yes, classic rookie mistake, will remove.